src/main/java/pl/wrapper/parking/facade/domain/ParkingController.java

```diff
@@ -1,13 +1,39 @@
 package pl.wrapper.parking.facade.domain;
 
+import static pl.wrapper.parking.infrastructure.error.HandleResult.handleResult;
+
 import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 import pl.wrapper.parking.facade.ParkingService;
+import pl.wrapper.parking.infrastructure.error.Result;
+import pl.wrapper.parking.pwrResponseHandler.dto.ParkingResponse;
 
 @RequestMapping("v1")
 @RestController
 @RequiredArgsConstructor
 class ParkingController {
     private final ParkingService parkingService;
+
+    @GetMapping("/name/{name}")
+    public ResponseEntity<String> getParkingByName(@PathVariable String name) {
```

Comment:
@PathVariable -> @RequestParam. Let's move the variable to the parameter section, it seems more appropriate as we are querying based on, well, parameters. Add an extra parameter with bool value named something along the lines of "opened". If false/null, do nothing, if true - return only if currently opened.